boms/bom/pom.xml

```diff
@@ -87,6 +87,7 @@
                            <licensesTemplate>${project.parent.build.directory}/licenses-project-template.xml</licensesTemplate>
                            <downloadSources>true</downloadSources>
                            <downloadPoms>true</downloadPoms>
+                           <analyzeRuntimeDependencies>true</analyzeRuntimeDependencies>
```

Comment:
Is it only added to the main BOM because you're still testing, or because that's the only BOM present in -prod?

Reply:
Because it's the only one used in prod.